LanguageFeatures/primary-constructors/static_processing_A06_t02.dart

```diff
@@ -29,56 +29,64 @@
 
 import '../../Utils/expect.dart';
 
-extension type ET1(String x) {
-  String Function() captureAtDeclaration = () => x;
-  String Function() captureInInitializer;
+String log = "";
 
-  this : captureInInitializer = (() => x);
+extension type ET1(String x) {
+  this : assert("$log=$x".isNotEmpty);
 }
 
 extension type ET2([String x = "default"]) {
-  String Function() captureAtDeclaration = () => x;
-  String Function() captureInInitializer;
-
-  this : captureInInitializer = (() => x);
+  this : assert("$log=$x".isNotEmpty);
 }
 
 extension type ET3({String x = "default"}) {
-  String Function() captureAtDeclaration = () => x;
-  String Function() captureInInitializer;
-
-  this : captureInInitializer = (() => x);
+  this : assert("$log=$x".isNotEmpty);
 }
 
 extension type ET4({required String x}) {
-  String Function() captureAtDeclaration = () => x;
-  String Function() captureInInitializer;
-
-  this : captureInInitializer = (() => x);
+  this : assert("$log=$x".isNotEmpty);
 }
 
 main() {
   var et1 = ET1("parameter");
-  Expect.equals("parameter", et1.captureAtDeclaration());
-  Expect.equals("parameter", et1.captureInInitializer());
+  Expect.equals("parameter", et1.x);
+  if (log.isNotEmpty) {
+    Expect.equals("parameter", log);
+    log = "";
+  }
```

Comment:
Why isn't this sufficient?

```suggestion
  Expect.equals("parameter", log);
  log = "";
```

Reply:
This handles the case when the test is run in release mode with assertions turned off. In that case, `log` will always be empty.